docs/src/main/paradox/scala/http/common/xml-support.md

```diff
@@ -4,23 +4,86 @@ Akka HTTP's @ref[marshalling](marshalling.md) and @ref[unmarshalling](unmarshall
 infrastructure makes it rather easy to seamlessly support specific wire representations of your data objects, like JSON,
 XML or even binary encodings.
 
-For XML Akka HTTP currently provides support for [Scala XML](https://github.com/scala/scala-xml) right out of the box through it's
+@@@ div { .group-java }
+
+Akka HTTP does not currently provide a Java API for XML support. If you need to
```

Comment:
Nice, that's a good start as long we don't support XML in Java in another way.

Reply:
Yes, it would be easy to roll into `akka-http-jackson` at the cost of an extra dependency, but I didn't want to open that can of worms in this PR. ;-)

Reply:
Yeah, good to not open that can yet ;)